lib/matplotlib/tests/test_text.py

```diff
@@ -886,3 +886,27 @@ def call(*args, **kwargs):
     # Every string gets a miss for the first layouting (extents), then a hit
     # when drawing, but "foo\nbar" gets two hits as it's drawn twice.
     assert info.hits > info.misses
+
+
+def test_annotate_offset_fontsize():
+    # Test that offset_fontsize parameter works and uses accurate values
+    fig, ax = plt.subplots()
+    # 10 points should be equal to 1 fontsize unit at fontsize=10
+    text_coords = ['offset points', 'offset fontsize']
+    xy_text = [(10, 10), (1, 1)]
+    index = 0
+    points_coords = ''
+    fontsize_coords = ''
+    for i in range(2):
+        ann = ax.annotate('test', xy=(0.5, 0.5),
+                          xytext=(xy_text[index]),
+                          fontsize='10',
+                          xycoords='data',
+                          textcoords=text_coords[index])
+        if i == 0:
+            points_coords = str(ann.get_window_extent())
+        else:
+            fontsize_coords = str(ann.get_window_extent())
```

Comment:
would spell the whole thing
```python
anns = [ax.annotate(...) for i in ...]
point_coords, fontsize_coords = [ann.get_window_extent() for ann in anns]
assert str(point_coords) == str(fontsize_coords)
```
but either way works.

Reply:
If you're happy with that, I'll squash it down 👍